src/sections/forms/contactDetails/index.js

```diff
@@ -0,0 +1,186 @@
+                    <div className="rainbow-flex  rainbow-justify rainbow-p-horizontal_small">
+                        <Field
+                            className="rainbow-m-top_small rainbow-rainbow-forms_inputs-field"
+                            icon={<Phone />}
+                            component={Input}
+                            name="phone"
+                            label="Phone Number"
+                            defaultMessage="Phone Number"
+                            placeholder="Enter your Phone Number"
+                            type="text" />
+                        <Field
+                            className="rainbow-m-top_small rainbow-rainbow-forms_inputs-field"
+                            icon={<Company />}
+                            component={Input}
+                            name="company"
+                            label="Company"
+                            defaultMessage="Company"
+                            placeholder="Enter your Company Name"
+                            type="text" />
+                    </div>
+                    <div className="rainbow-flex  rainbow-justify rainbow-p-horizontal_small">
+                        <Field
+                            className="rainbow-m-top_small rainbow-rainbow-forms_inputs-field"
+                            icon={<SignPost />}
+                            component={Input}
+                            name="address"
+                            label="Address"
+                            defaultMessage="Address"
+                            placeholder="Enter your Address"
+                            type="text" />
+                    </div>
+                    <div className="rainbow-flex rainbow-justify rainbow-p-horizontal_small">
+                        <Field
+                            className="rainbow-m-top_small rainbow-rainbow-forms_inputs-field"
+                            icon={<City />}
+                            component={Input}
+                            name="city"
+                            label="City"
+                            defaultMessage="City"
+                            placeholder="City"
+                            type="text" />
+                        <Field
+                            className="rainbow-m-top_small rainbow-rainbow-forms_inputs-field"
+                            icon={<Map />}
+                            component={Input}
+                            name="postCode"
+                            label="Postal Code"
+                            defaultMessage="Postal Code"
+                            placeholder="Postal Code"
+                            type="text" />
+                        <Field
+                            className="rainbow-m-top_small rainbow-rainbow-forms_inputs-field"
+                            component={Select}
+                            options={[{ value: 'US', label: 'US' }, { value: 'Canada', label: 'Canada' }]}
```

Comment:
put this above the component, let's try not mix js with jsx:
const COUNTRY_OPTIONS = [{ value: 'US', label: 'US' }, { value: 'Canada', label: 'Canada' }];
also this have performance considerations since here a new array is create in every rerender